src/api/routes/user.js

```diff
@@ -15,4 +15,6 @@ router.get("/:user_id/profile", checkAuth, UserController.getProfile);
 
 router.patch("/me/change_password", checkAuth, UserController.changePassword);
 
+router.put("/me/photo", checkAuth, UserController.changeProfilePhoto);
```

Comment:
:warning: **You haven't created the `changeProfilePhoto` function and this is causing the server to crash on startup!** :warning: 

(I also think `changePhoto` will be better since it's shorter)